src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
According to the task: half of the figures should be the default, and others - be random figures. 
![image](https://user-images.githubusercontent.com/32339836/218494974-5f519aec-5e52-4a0b-b171-9c17c4ddbdb8.png)